src/main/java/org/gridsuite/modification/server/NetworkModificationController.java

```diff
@@ -178,14 +177,8 @@ public ResponseEntity<Void> stopBuild(@Parameter(description = "Build receiver")
     @GetMapping(value = "/network-modifications/catalog/line_types", produces = MediaType.APPLICATION_JSON_VALUE)
     @Operation(summary = "Get a line types catalog")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The line types catalog is returned")})
-    public ResponseEntity<List<LineType>> getLineTypesCatalog(@Parameter(description = "type") @RequestParam(name = "category", required = false) LineTypeCategory category) {
-        List<LineType> res;
-        if (category != null) {
-            res = lineTypesCatalogService.getLineTypesCatalog(category);
-        } else {
-            res = lineTypesCatalogService.getAllLineTypesCatalog();
-        }
-        return ResponseEntity.ok().body(res);
+    public ResponseEntity<List<LineType>> getLineTypesCatalog() {
```

Comment:
`getLinesTypes` ?
We return a list not a catalog DTO

Reply:
renamed